fuse_core/include/fuse_core/eigen.h

```diff
@@ -36,6 +36,8 @@
 
 #include <Eigen/Core>
 
+#include <sstream>
```

Comment:
[fuse_core:make] /home/swilliams/locus_dev/src/fuse/fuse_core/include/fuse_core/eigen.h:72:  Add #include <string> for string  [build/include_what_you_use] [4]

```suggestion
#include <sstream>
#include <string>
```

Reply:
Done